components/drafts/draft_title/index.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getUser, getUserIdsInChannels} from 'mattermost-redux/selectors/entities/users';
+import {getUserIdFromChannelName} from 'mattermost-redux/utils/channel_utils';
+import {getTeammateNameDisplaySetting} from 'mattermost-redux/selectors/entities/preferences';
+import {displayUsername} from 'mattermost-redux/utils/user_utils';
+import {Constants} from 'utils/constants';
+
+import {Channel} from 'mattermost-redux/types/channels';
+import {GlobalState} from 'types/store';
+
+import DraftTitle from './draft_title';
+
+type OwnProps = {
+    channel: Channel;
```

Comment:
Probably over optimizing, but here you only seem to need the display name, name and id of the channel.